src/mokka/src/main/java/pl/hycom/mokka/emulator/mock/MockConfigurationManager.java

```diff
@@ -189,7 +233,14 @@ public boolean setEnabled(Long id, boolean enabled) {
 
         Optional<MockConfiguration> mock = repository.findById(id);
         mock.ifPresent(m -> m.setEnabled(enabled));
-        mock.ifPresent(this::saveOrUpdateMockConfiguration);
+
+        if(enabled){
+            StubMappingEnabledEvent stubMappingEnabledEvent = new StubMappingEnabledEvent(this, id);
+            applicationEventPublisher.publishEvent(stubMappingEnabledEvent);
+        } else {
+            StubMappingDisabledEvent stubMappingDisabledEvent = new StubMappingDisabledEvent(this, id);
+            applicationEventPublisher.publishEvent(stubMappingDisabledEvent);
+        }
```

Comment:
Would be wiser to split `setEnabled` and expose two separate methods instead: enable(id), disable(id)